pkg/vpcmodel/textOutput.go

```diff
@@ -40,13 +41,28 @@ func headerOfAnalyzedVPC(uc OutputUseCase, vpcName, vpc2Name string, c1 *VPCConf
 		if explanation.connQuery != nil {
 			connStr = " for " + explanation.connQuery.String()
 		}
-		return fmt.Sprintf("Connectivity explanation%s between %s and "+
-			"%s\n===================================================================================================\n\n",
-			connStr, explanation.src, explanation.dst), nil
+		srcNetworkInterfaces := explainNetworkInterfaces(explanation.srcNetworkInterfacesFromIP)
+		dstNetworkInterfaces := explainNetworkInterfaces(explanation.dstNetworkInterfacesFromIP)
+		header1 := fmt.Sprintf("Connectivity explanation%s between %s%s and %s%s",
+			connStr, explanation.src, srcNetworkInterfaces, explanation.dst, dstNetworkInterfaces)
+		header2 := strings.Repeat("=", len(header1))
+		return header1 + "\n" + header2 + "\n\n", nil
```

Comment:
this should be a method of `Explanation` ?

Reply:
I don't think that "pure" printing related functions should reside in `Explanation`.

Do you think `listNetworkInterfaces` (perhaps without the "(" + ") ) can be useful in other cases? perhaps this function can be moved to a common file? Or maybe to Explanation?

Reply:
I would move this code (under `case Explain`) to a function such as `explainabilityReportHeader(explanation *Explanation)`, and maybe have both this new function and `explainNetworkInterfaces` in a file related to explainability.

Reply:
I've added a file for printing related to explanation, and moved other relevant functions